### PR TITLE
Add blog article: It is not a choice of the messenger

### PR DESCRIPTION
**Pull request by Echolon**
I wrote this comment on the current situation in the media. As you can read, my focus is the decision making for people.

I wrote a general article, but added also direct example or "statements" within the article. In the end I also provided some links for further reading.

Please review, but please consider that I invested several hours in this incl. a [Pull Request](https://github.com/xsf/xmpp.org/pull/870) for improving the XSF Client page. Please provide further links if you think it suits!

Tweet etc. will follow.

### CODE REVIEW

File: content/posts/blog/2021-01-20.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+hand, I also see they usually do not switch because they made a very evaluated choice for one specific messenger.
+
+Actually, most will just yet add another messenger to their list of messengers. Likely, that is more frustrating than helping, which brings me 
+back to the beginning and the actual question: What issue or what problem does one want to solve here? What are the incentives? Can one decide 
+towards a solution that, when the next hype chases through the internet, one will have a better basis?
+
+A possible background of that question may be different expectations of some functionality, the basic requirement of effective data privacy and 
+sovereignty or simply the possibility to reach all my contacts. However, that change will still require leaving behind the previous messaging 
+system and switching to a new one, which almost always results in the loss of the existing contact connections. Furthermore, in terms of privacy, 
+one is almost always is required to provide a phone number. Again, what does one try to solve?
```
Comment (wurstsalat3000) [inline content/posts/blog/2021-01-20.md:24]: one is almost always is required to -> one is almost always required to

File: content/posts/blog/2021-01-20.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+### Sovereignty of Communication
+
+So, as you likely already wondered, why am I going to advocate for XMPP now? I think the choice one should do firsthand, is not a choice of the 
+messenger you use, but of the underlying software technology. Obviously, one should actually always do that. So, here we are: Consider the choice 
+of technology first before you jump from one recommendation to the next one.
+
+> XMPP is an open protocol, as HTTP is for websites. It mostly does not matter how your website look like, but everyone can interact with it. This
```
Comment (wurstsalat3000) [inline content/posts/blog/2021-01-20.md:32]: your website **looks** like

File: content/posts/blog/2021-01-20.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+there are:
+
+- Decentralization (federation) of communication services
+- Standardization and extensibility of technology
+- Innovation and the use of open development
+- Privacy and control, also by trustful encryption
+- Enabling of free speech and freedom of software
+
+> XMPP provides defined information how to handle the communication in a network. The user decides which messaging software suits or where the 
+data should stored.
```
Comment (wurstsalat3000) [inline content/posts/blog/2021-01-20.md:47]: should **be** stored

File: content/posts/blog/2021-01-20.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+In contrast to the current situation of many people suggesting one specific messenger, in the XMPP universe you have the choice between many 
+messengers – pick the one you like and not what others think you like until the next hype starts. The difference is that you still have the comfort to be 
+in the same network, independent of your choice today. I think this is a real solution.
```
Comment (wurstsalat3000) [inline content/posts/blog/2021-01-20.md:51]: I think there is an unwanted linebreak here

File: content/posts/blog/2021-01-20.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+### It is not a choice of the messenger, but of the technology
+
+Consider a decision on the technology you intend to use, then decide which messenger suits you or your environment. XMPP and its community have a long-lasting
```
Comment (wurstsalat3000) [inline content/posts/blog/2021-01-20.md:55]: More unwanted linebreaks in this paragraph?

File: content/posts/blog/2021-01-20.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+experience, which has been successfully utilized in a lot of software. We think this was and still is a good solution for the entire internet and most people 
+and organizations out there – and if not yet, it is open for your modification in the future - and if not yet, it is open for you to make modifications in the 
+future to make it do what you need.
+
+> XMPP has not just a wide range of messaging software; there is also various server software and the tools (libraries) to build infrastructure yourself.
+
+There is not one single solution how communication is designed. Make a choice of the technology which is a persistent and sustainable decision. For that 
+purpose and many more we recommend the use of XMPP – the Extensible Messaging and Presence Protocol.
+
+If you are interested in making a choice for the next decade, I added further information below.
```
Comment (wurstsalat3000) [inline content/posts/blog/2021-01-20.md:65]: I added further information below -> let me point you to these further readings

**Review by wurstsalat3000 — COMMENTED** [review]
Well done! :)

File: content/posts/blog/2021-01-20.md (hunk 5 above)
Comment (Echolon) [reply content/posts/blog/2021-01-20.md:55]: Dunno, just Github / Texteditor things. I removed them

File: content/posts/blog/2021-01-20.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+## It is not a choice of the messenger
+
+Recently, several people reached out to me asking what kind of messenger they should be using now. Furthermore, they said that they actually 
+do not understand what is going on and whether they should switch from one of the commonly known messengers to another one. 
+I was wondering how to answer this. Obviously, I could have stood-up and just advocated for XMPP (Extensible Messaging and Presence Protocol), 
+but then I thought this alone is not a helpful answer. On the one hand, I see that many people just make a short-term decision. On the other 
+hand, I also see they usually do not switch because they made a very evaluated choice for one specific messenger.
+
+Actually, most will just yet add another messenger to their list of messengers. Likely, that is more frustrating than helping, which brings me
```
Comment (ge0rg) [inline content/posts/blog/2021-01-20.md]: s/yet// maybe?

File: content/posts/blog/2021-01-20.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+blog_id: blog
+category: misc
+---
+
+## It is not a choice of the messenger
+
+Recently, several people reached out to me asking what kind of messenger they should be using now. Furthermore, they said that they actually do not understand what is going on and whether they should switch from one of the commonly known messengers to another one. 
+I was wondering how to answer this. Obviously, I could have stood-up and just advocated for XMPP (Extensible Messaging and Presence Protocol), but then I thought this alone is not a helpful answer. On the one hand, I see that many people just make a short-term decision. On the other hand, I also see they usually do not switch because they made a very evaluated choice for one specific messenger.
+
+Actually, most will just yet add another messenger to their list of messengers. Likely, that is more frustrating than helping, which brings me back to the beginning and the actual question: What issue or what problem does one want to solve here? What are the incentives? Can one decide towards a solution that, when the next hype chases through the internet, one will have a better basis?
```
Comment (ge0rg) [inline content/posts/blog/2021-01-20.md:14]: s/yet//

s/one will/will/

File: content/posts/blog/2021-01-20.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+
+- Decentralization (federation) of communication services
+- Standardization and extensibility of technology
+- Innovation and the use of open development
+- Privacy and control, also by trustful encryption
+- Enabling of free speech and freedom of software
+
+> XMPP provides defined information how to handle the communication in a network. The user decides which messaging software suits or where the data should be stored.
+
+In contrast to the current situation of many people suggesting one specific messenger, in the XMPP universe you have the choice between many messengers – pick the one you like and not what others think you like until the next hype starts. The difference is that you still have the comfort to be in the same network, independent of your choice today. I think this is a real solution.
```
Comment (ge0rg) [inline content/posts/blog/2021-01-20.md:35]: s/what others think you like/what others like (and *maybe* think that you might like as well)/

Comment (Echolon) [reply content/posts/blog/2021-01-20.md:35]: Thanks - everything is resolved!

File: content/posts/blog/2021-01-20.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
+---
+title: It is not a choice of the messenger
```
Comment (SamWhited) [inline content/posts/blog/2021-01-20.md:2]: I'm not actually sure what this title means, but I don't think it translated to English very well. What is a messenger in this context? An individual client? Maybe "Choose networks, not apps" or something to that effect?

File: content/posts/blog/2021-01-20.md [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+date: 2021-01-20 06:30
+author: Edward Maurer
+blog_id: blog
+category: misc
+---
+
+## It is not a choice of the messenger
+
+Recently, several people reached out to me asking what kind of messenger they should be using now. Furthermore, they said that they actually do not understand what is going on and whether they should switch from one of the commonly known messengers to another one.
```
[the remark this answers is not in the document]
Comment (SamWhited) [reply content/posts/blog/2021-01-20.md:11]: Remove the "one" from the end. "from one of the commonly known messengers to another" is fine.

File: content/posts/blog/2021-01-20.md [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+date: 2021-01-20 06:30
+author: Edward Maurer
+blog_id: blog
+category: misc
+---
+
+## It is not a choice of the messenger
+
+Recently, several people reached out to me asking what kind of messenger they should be using now. Furthermore, they said that they actually do not understand what is going on and whether they should switch from one of the commonly known messengers to another one. 
+I was wondering how to answer this. Obviously, I could have stood-up and just advocated for XMPP (Extensible Messaging and Presence Protocol), but then I thought this alone is not a helpful answer. On the one hand, I see that many people just make a short-term decision. On the other hand, I also see they usually do not switch because they made a very evaluated choice for one specific messenger.
```
Comment (SamWhited) [inline content/posts/blog/2021-01-20.md:12]: I'd remove either the "just" or the "stood-up and". "I could have just advocated for XMPP" or "I could have stood-up and advocated for XMPP".

Comment (SamWhited) [reply content/posts/blog/2021-01-20.md:12]: If it's phrased this way you need a comma after "thought" and quotations around "this alone is not a helpful answer": 'then I thought, "this alone is not a helpful answer"'. Personally I would rephrase so that this isn't necessary.

Also, why is it not a helpful answer, and what does it mean to "only make a short-term decision"? I don't fully understand what this paragraph is trying to say.

File: content/posts/blog/2021-01-20.md [hunk 13]
```diff
@@ -0,0 +1,55 @@
...
+
+Actually, most will just add another messenger to their list of messengers. Likely, that is more frustrating than helping, which brings me back to the beginning and the actual question: What issue or what problem does one want to solve here? What are the incentives? Can one decide towards a solution that, when the next hype chases through the internet, will have a better basis?
```
Comment (SamWhited) [inline content/posts/blog/2021-01-20.md:14]: Remove the "Actually,".  
The last sentence on this line has some grammatical problems and I'm not really sure what it's trying to say. Maybe "Can one find a solution that avoids hype and doesn't involve installing more than one messenger app?" hopefully that's saying the same thing, apologies if not. What is a "better basis"? The basis of what?

File: content/posts/blog/2021-01-20.md [hunk 14]
```diff
@@ -0,0 +1,55 @@
...
+
+A possible background of that question may be different expectations of some functionality, the basic requirement of effective data privacy and sovereignty or simply the possibility to reach all my contacts. However, that change will still require leaving behind the previous messaging system and switching to a new one, which almost always results in the loss of the existing contact connections. Furthermore, in terms of privacy, one is almost always required to provide a phone number. Again, what does one will try to solve?
```
Comment (SamWhited) [inline content/posts/blog/2021-01-20.md:16]: I'm also not sure what the "background" of a question means but "A possible background of that question" sounds wrong to me. My apologies for not being able to suggest a good replacement here.

I know what you mean by "contact connections" but the phrasing feels awkward. Maybe just "existing contacts"?

It also feels awkward again to ask what question we're trying to answer at the end. Isn't this paragraph explaining the problem we're trying to solve?

File: content/posts/blog/2021-01-20.md [hunk 15]
```diff
@@ -0,0 +1,55 @@
...
+
+### Sovereignty of Communication
+
+So, as you likely already wondered, why am I going to advocate for XMPP now? I think the choice one should do firsthand, is not a choice of the messenger you use, but of the underlying software technology. Obviously, one should actually always do that. So, here we are: Consider the choice of technology first before you jump from one recommendation to the next one.
```
Comment (SamWhited) [inline content/posts/blog/2021-01-20.md:20]: The first sentence here is a bit hard to read. Maybe simplify it to something like "So, as you might have suspected, now I'm going to advocate for XMPP."

File: content/posts/blog/2021-01-20.md [hunk 16]
```diff
@@ -0,0 +1,55 @@
...
+
+> XMPP is an open protocol, as HTTP is for websites. It mostly does not matter how your website looks like, but everyone can interact with it. This principle is the idea behind XMPP for instant messaging.
```
Comment (SamWhited) [inline content/posts/blog/2021-01-20.md:22]: Is this a quote? Where does it come from? If it's a quote I wouldn't use it because the grammar isn't great. If it's not I'd remove the quote formatting and change it to something like:

> XMPP is an open protocol, like HTTP is for the web. It doesn't matter what your website looks like, everyone can interact with it. This principle is the idea behind XMPP, but for instant messaging.

File: content/posts/blog/2021-01-20.md [hunk 17]
```diff
@@ -0,0 +1,55 @@
...
+
+We at the XMPP Standards Foundation, but also many others involved in this open technology, think that XMPP is a good choice as a communication technology when it comes to this question. XMPP as a protocol has been around for about two decades since its creation in 1999 and a lot of experience has been gathered during that period. Mainly, this includes several important key indicators for the choice of technology and enabling of sovereignty as
```
Comment (SamWhited) [inline content/posts/blog/2021-01-20.md:24]: Remove "when it comes to this question" and end the sentence after "technology". We think it's good in general, not just with respect to data privacy :)

You mention data sovereignty here again, but it wasn't really ever defined. Maybe define it the first time you mention it? I can take a guess, but I'm not really sure what that means.

File: content/posts/blog/2021-01-20.md [hunk 18]
```diff
@@ -0,0 +1,55 @@
...
+
+### Sovereignty of Communication
+
+So, as you likely already wondered, why am I going to advocate for XMPP now? I think the choice one should do firsthand, is not a choice of the messenger you use, but of the underlying software technology. Obviously, one should actually always do that. So, here we are: Consider the choice of technology first before you jump from one recommendation to the next one.
+
+> XMPP is an open protocol, as HTTP is for websites. It mostly does not matter how your website looks like, but everyone can interact with it. This principle is the idea behind XMPP for instant messaging.
+
+We at the XMPP Standards Foundation, but also many others involved in this open technology, think that XMPP is a good choice as a communication technology when it comes to this question. XMPP as a protocol has been around for about two decades since its creation in 1999 and a lot of experience has been gathered during that period. Mainly, this includes several important key indicators for the choice of technology and enabling of sovereignty as 
+there are:
```
Comment (SamWhited) [inline content/posts/blog/2021-01-20.md:25]: s/there are/it supports/ maybe?

File: content/posts/blog/2021-01-20.md [hunk 19]
```diff
@@ -0,0 +1,55 @@
...
+
+- Decentralization (federation) of communication services
+- Standardization and extensibility of technology
+- Innovation and the use of open development
+- Privacy and control, also by trustful encryption
```
Comment (SamWhited) [inline content/posts/blog/2021-01-20.md:30]: I would remove "also by trustful encryption" personally and just list the thing you're saying is good, not the mechanism by which it is achieved. However, if you do want to keep it, maybe change it to ", using encryption" or something like that.

File: content/posts/blog/2021-01-20.md [hunk 20]
```diff
@@ -0,0 +1,55 @@
...
+- Enabling of free speech and freedom of software
```
Comment (SamWhited) [inline content/posts/blog/2021-01-20.md:31]: I would remove this last bullet if it were me, "free speech" and "freedom of software" are vaguely defined topics that people love to have hand wavy arguments about.

Also in the US right now given the recent bans of hate speech websites being in the news adding this bullet may read as a coded message for those people to come to XMPP since right wing extremist groups often co-opt the language of the left and use "free speech" to mean "I want to spout hate speech on the internet and everyone bans me for it", and we don't want to be sending that message unintentionally. Of course, when this is out of the news this isn't an association I'd normally make, but at this time I'd argue we shouldn't make it sound like we're advocating for their so-called "free speech" platforms (which have nothing to do with the actual concept of free speech).

File: content/posts/blog/2021-01-20.md [hunk 21]
```diff
@@ -0,0 +1,55 @@
...
+
+> XMPP provides defined information how to handle the communication in a network. The user decides which messaging software suits or where the data should be stored.
+
+In contrast to the current situation of many people suggesting one specific messenger, in the XMPP universe you have the choice between many messengers – pick the one you like and not what others like (and maybe think that you might like as well) until the next hype starts. The difference is that you still have the comfort to be in the same network, independent of your choice today. I think this is a real solution.
```
Comment (SamWhited) [inline content/posts/blog/2021-01-20.md:35]: "you have **a** choice between many messengers"

"until the next internet trend", maybe.

File: content/posts/blog/2021-01-20.md [hunk 22]
```diff
@@ -0,0 +1,55 @@
...
+
+### It is not a choice of the messenger, but of the technology
+
+Consider a decision on the technology you intend to use, then decide which messenger suits you or your environment. XMPP and its community have a long-lasting experience, which has been successfully utilized in a lot of software. We think this was and still is a good solution for the entire internet and most people and organizations out there – and if not yet, it is open for you to make modifications in the future to make it do what you need.
```
Comment (SamWhited) [inline content/posts/blog/2021-01-20.md:39]: "it is open for you to make modifications in the future to make it do what you need"

Personally I always hate it when people say this. I don't want to do a bunch of work, I want to pick an off the shelf solution and get back to talking to my friends or growing my business or whatever I'm using chat for. When someone tells me "well you can fix it to do what you want" I assume that means it's not very good to start with and they just want free labor. But YMMV.

File: content/posts/blog/2021-01-20.md [hunk 23]
```diff
@@ -0,0 +1,55 @@
...
+
+> XMPP has not just a wide range of messaging software; there is also various server software and the tools (libraries) to build infrastructure yourself.
+
+There is not one single solution how communication is designed. Make a choice of the technology which is a persistent and sustainable decision. For that purpose and many more we recommend the use of XMPP – the Extensible Messaging and Presence Protocol.
```
Comment (SamWhited) [inline content/posts/blog/2021-01-20.md:43]: "There is not a single way in which communication apps are designed" maybe.

Comment (SamWhited) [inline content/posts/blog/2021-01-20.md:43]: "Make a choice to use technology that is persistent and sustainable." Maybe, although I'm not sure why it's "persistent"?

**Review by SamWhited — COMMENTED** [review]
Thanks for writing an article; great job! I've left a bunch of comments on grammar and a few places I wasn't sure what the argument was or thought the argument might be other than you intended. Sorry for the wall of text, and thanks again!

File: content/posts/blog/2021-01-20.md (hunk 10 above)
Comment (Echolon) [reply content/posts/blog/2021-01-20.md:2]: I think about 'chat messenger' now?

Comment (SamWhited) [reply content/posts/blog/2021-01-20.md:2]: I don't think "chat messenger" is a thing either, and I'm not sure what that means. Do you mean an instant messaging app?

File: content/posts/blog/2021-01-20.md [hunk 24]
```diff
@@ -0,0 +1,55 @@
...
+
+A possible background of that question may be different expectations of some functionality, the basic requirement of effective data privacy and sovereignty or simply the possibility to reach all my contacts. However, that change will still require leaving behind the previous messaging system and switching to a new one, which almost always results in the loss of the existing contact connections. Furthermore, in terms of privacy, one is almost always required to provide a phone number. Again, what does one try to solve?
+
+### Sovereignty of Communication
+
+So, as you likely already wondered, why am I going to advocate for XMPP now? I think the choice one should do firsthand, is not a choice of the messenger you use, but of the underlying software technology. Obviously, one should actually always do that. So, here we are: Consider the choice of technology first before you jump from one recommendation to the next one.
```
[the remark this answers is not in the document]
Comment (Echolon) [reply content/posts/blog/2021-01-20.md:20]: good point

File: content/posts/blog/2021-01-20.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (Echolon) [reply content/posts/blog/2021-01-20.md:11]: > Furthermore, they said that they actually do not understand **what they should be concerned about** and whether they should switch from one of the commonly known messengers to another.

File: content/posts/blog/2021-01-20.md (hunk 10 above)
Comment (Echolon) [reply content/posts/blog/2021-01-20.md:2]: > It is not a choice of the instant messaging app

File: content/posts/blog/2021-01-20.md (hunk 12 above)
Comment (Echolon) [reply content/posts/blog/2021-01-20.md:12]: I intended to say, that people dont think a lot about where to go next. They might just do it, then wonder why it was not a sufficient decision. I also not just wanted to simply argue why XMPP is good, I also wanted to show the general reasons behind it etc.

Maybe that improves a bit:
> I was wondering how to answer this. Obviously, I could have stood-up and advocated for XMPP (Extensible Messaging and Presence Protocol), but then I thought this alone is not a helpful answer. On the one hand, I see that many people just make a short-term decision about their communication software. On the other hand, I also see they usually do not switch the platform because they made a very evaluated choice for one specific messenger. Many just switch, again and again.

File: content/posts/blog/2021-01-20.md (hunk 20 above)
Comment (Echolon) [reply content/posts/blog/2021-01-20.md:31]: I think switch to `Enabling more sovereignty of communication` is better, at least I want to make a statement that you gain back control in this direction (for the good or the bad)

File: content/posts/blog/2021-01-20.md (hunk 21 above)
Comment (Echolon) [reply content/posts/blog/2021-01-20.md:35]: yes!

File: content/posts/blog/2021-01-20.md (hunk 22 above)
Comment (Echolon) [reply content/posts/blog/2021-01-20.md:39]: >  We think this was and still is a good solution for the entire internet and most people and organizations out there – and it is open for you to make innovations for the future.

File: content/posts/blog/2021-01-20.md (hunk 23 above)
Comment (Echolon) [reply content/posts/blog/2021-01-20.md:43]: Yes

Comment (Echolon) [reply content/posts/blog/2021-01-20.md:43]: I removed it and pointed out the standardization

> Make a choice of standardized technology which is a sustainable decision.

File: content/posts/blog/2021-01-20.md (hunk 13 above)
Comment (Echolon) [reply content/posts/blog/2021-01-20.md:14]: > Can one find a solution that avoids hype and does not involve installing more than one messenger app that will have a better technological basis?

File: content/posts/blog/2021-01-20.md (hunk 16 above)
Comment (Echolon) [reply content/posts/blog/2021-01-20.md:22]: I placed those as better examples within the actual text, some were recommending this to be more clear to the user. I hope Pelican will show it better

File: content/posts/blog/2021-01-20.md (hunk 12 above)
Comment (Echolon) [reply content/posts/blog/2021-01-20.md:12]: > Many people just switch again and again.

**Comment by Echolon** [comment]
I will just merge it now as it has been review by many people, also at least one native speaker.

Lets go! (finally :smile: )

Thanks to:

@ge0rg @mdosch @vanitasvitae @wurstsalat3000 @SamWhited mathieui for reviewing and also my personal friends doing the first raw checks!

XMPP For The Win! :heart:

**Comment by SamWhited** [comment]
I'd get someone who has merge permissions and is a native speaker to do an editing pass before hitting merge if I were you. I tried to catch a lot of stuff, but there was a lot more I wasn't sure about and no one has reviewed the new fixes. I don't know what XSF policy is here though, but there's still a lot of weird grammar in this post.

File: content/posts/blog/2021-01-20.md [hunk 25]
```diff
@@ -0,0 +1,53 @@
...
+date: 2021-01-20 06:30
+author: Edward Maurer
+blog_id: blog
+category: misc
+---
+
+## It is not a choice of the instant messaging app
+
+Recently, several people reached out to me asking what kind of messenger they should be using now. Furthermore, they said that they actually do not understand what they should be concerned about and whether they should switch from one of the commonly known messengers to another. 
+I was wondering how to answer this. Obviously, I could have stood-up and advocated for XMPP (Extensible Messaging and Presence Protocol), but then I thought this alone is not a helpful answer. On the one hand, I see that many people just make a short-term decision about their communication software. On the other hand, I also see they usually do not switch the platform because they made a very evaluated choice for one specific messenger. Many people just switch again and again.
```
Comment (SamWhited) [inline content/posts/blog/2021-01-20.md:12]: nit: no hyphen in "stood-up", sorry I didn't catch that last time

File: content/posts/blog/2021-01-20.md [hunk 26]
```diff
@@ -0,0 +1,53 @@
...
+
+Actually, most will just add yet another messenger to their list of messengers. Likely, that is more frustrating than helping, which brings me back to the beginning and the actual question: What issue or what problem does one want to solve here? What are the incentives? Can one find a solution that avoids hype and does not involve installing more than one messenger app that will have a better technological basis?
```
Comment (SamWhited) [inline content/posts/blog/2021-01-20.md:14]: "more frustrating than helpful"

File: content/posts/blog/2021-01-20.md [hunk 27]
```diff
@@ -0,0 +1,53 @@
...
+
+### It is not a choice of the instant messaging app, but of the technology
+
+Consider a decision on the technology you intend to use, then decide which messenger suits you or your environment. XMPP and its community have a long-lasting experience, which has been successfully utilized in a lot of software. We think this was and still is a good solution for the entire internet and most people and organizations out there – and it is open for you to make innovations for the future.
+
+> XMPP has not just a wide range of messaging software; there is also various server software and the tools (libraries) to build infrastructure yourself.
+
+There is not a single way in which communication apps are designed. Make a choice of standardized technology which is a sustainable decision. For that purpose and many more we recommend the use of XMPP – the Extensible Messaging and Presence Protocol.
+
+If you are interested in making a choice for the next decade, let me point you to these further readings:
```
Comment (SamWhited) [inline content/posts/blog/2021-01-20.md:43]: "let me point you to some further reading:"

File: content/posts/blog/2021-01-20.md (hunk 25 above)
Comment (Zash) [inline content/posts/blog/2021-01-20.md:12]: > they said that they actually do not understand [...]

Howabout "that they don't actually understand [...]"

> On the one hand, [...]

I would have dropped the "the" there.

> they usually do not switch the platform because

I think this would be fine without the "the" here too

Comment (SamWhited) [reply content/posts/blog/2021-01-20.md:12]: >> On the one hand, [...]
> I would have dropped the "the" there.

"on the one hand" is correct, as far as I'm aware.

Comment (SamWhited) [reply content/posts/blog/2021-01-20.md:12]: As far as "platform" you need the "the" unless it's "platforms".

> they usually do not switch the platform

or 

> they usually do not switch platforms

I do think the second one is better personally.

Comment (Echolon) [reply content/posts/blog/2021-01-20.md:12]: @zash thank you!

**Comment by Echolon** [comment]
Thanks everyone for helping to make this great!
